docs/standard/native-interop/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Native interoperability
+description: Learn how to interface with native components in .NET.
+author: blackdwarf
+ms.author: ronpet
+ms.date: 06/20/2016
+ms.technology: dotnet-standard
+---
+# Native Interoperability
+
+In this section, we will dive a little bit deeper into all three ways of doing "native interoperability" that are available using .NET.
```

Comment:
In this section -> In this article?

We don't use we in content unless it's a recommendation so please rewrite this without using we.